### PR TITLE
Added 'excludeActions' prop to 'MenuColumn' component

### PR DESCRIPTION
**Pull request by gitrite**

### CODE REVIEW

File: packages/veritone-react-common/src/components/DataTable/table.test.js [hunk 1]
```diff
@@ -151,6 +151,25 @@ describe('MenuColumn', function() {
...
+          data={data}
+          dataKey="actions"
+          protectedActions={['delete']}
+          excludeActions={['alter']}
+        />
+      </SupressColumnWarnings>
+    );
+
+    const menuItems = wrapper.find('Menu').prop('children');
+    expect(map(menuItems, 'key')).not.toContain('alter');
```
Comment (olslash) [inline packages/veritone-react-common/src/components/DataTable/table.test.js]: lets put an affirmative test too -- verify that delete and manage show up.

File: packages/veritone-react-common/src/components/DataTable/table.test.js [hunk 2]
```diff
@@ -151,6 +151,25 @@ describe('MenuColumn', function() {
         .length
     ).toEqual(0);
   });
+
+  it('excludes specified actions', function() {
+    let data = {
+      actions: ['delete', 'alter', 'manage']
+    };
+    let wrapper = mount(
```
Comment (olslash) [inline packages/veritone-react-common/src/components/DataTable/table.test.js]: data, wrapper should be const

Comment (gitrite) [reply packages/veritone-react-common/src/components/DataTable/table.test.js]: fixed

File: packages/veritone-react-common/src/components/DataTable/table.test.js (hunk 1 above)
Comment (gitrite) [reply packages/veritone-react-common/src/components/DataTable/table.test.js]: added

**Pull request merged**